src/components/utils/index.js

```diff
@@ -52,31 +52,15 @@ export const getUnsupportedMobileBrowserError = () => {
 // Copied from https://github.com/muaz-khan/DetectRTC/blob/master/DetectRTC.js
 export const isDesktop = !(/Android|webOS|BB10|BlackBerry|IEMobile|Opera Mini|Mobile|mobile/i.test(navigator.userAgent || '')) && !isIOS
 
+const isWindows = (navigator.userAgent || "").includes('Windows')
+
+const maxTouchPoints = navigator.maxTouchPoints || navigator.msMaxTouchPoints
+const isTouchable = 'ontouchstart' in window
+  || maxTouchPoints > 0
+  || window.matchMedia && matchMedia('(any-pointer: coarse)').matches
+
 // To detect hybrid desktop/mobile devices which have a rear facing camera such as the Surface
-export async function isHybrid(facingMode = 'environment') {
-  return isDesktop &&
-    navigator.platform === 'Win32' &&
-    await navigator.mediaDevices.getUserMedia(
-      {
-        video: {
-          facingMode
-        }
-      }
-    ).then(async (mediaStream) => {
-      const devices = mediaStream.getTracks();
-      const matches = ['back', 'rear', 'world'];
-      const device = devices.find(d => matches.some(match => d.label.toLocaleLowerCase().includes(match)));
```

Comment:
One thing I'd like to verify before we remove this code is that there isn't any situation, most likely in react-webcam where we rely on enumerateDevices to grab a label that determines whether it is a back for front facing camera. The issue I ran into was that if you hadn't called getUserMedia, those labels would not have the relevant information. I looked at react-webcam, and it looks like it's undergone some significant changes since I last saw it, so maybe this won't be an issue, but I just want to check.

Reply:
Thanks Sky, yes, we changed the way we detect back camera on react-webcam and we tested the change on a couple of surface devices to make sure things work as expected. If you have some times would you mind testing on your surface and check if it all works as expected? The link is
https://1133-pr-onfido-sdk-ui-onfido.surge.sh?useLiveDocumentCapture=true